tests/test_rootfs.py

```diff
@@ -236,6 +236,30 @@ def test_expected_files_ext234_mender_configure(
             assert "Type: symlink" in output
             assert 'Fast link dest: "/data/mender-configure"' in output
 
+    @pytest.mark.only_with_image("ext4", "ext3", "ext2")
+    @pytest.mark.min_mender_version("3.1.0")
+    def test_expected_files_ext234_mender_monitor(
+        self, conversion, bitbake_path, bitbake_variables, latest_rootfs
+    ):
+        """Test mender-monitor expected files (only state folder)"""
+
+        if not conversion:
+            pytest.skip("Test only applicable for mender-convert images.")
+
+        with make_tempdir() as tmpdir:
+            # Check whether mender-monitor exists in /var/lib
+            self.verify_file_exists(
+                tmpdir, latest_rootfs, "/var/lib", "mender-monitor",
```

Comment:
at this point it is a link? should we check it as well? (feel free to ignore I am not an expert on those tests)

Reply:
We check that it is a link just few lines below

Reply:
I saw it, but I thought it is different place since it uses debugfs. ok then. thank you.